src/components/features/Feedback/Feedback.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styles from './Feedback.module.scss';
+
+const Feedback = () => {
+  return (
+    <div className={styles.root}>
+      <div className='container'>
+        <div className={styles.panelBar}>
+          <div className='row no-gutters align-items-end'>
+            <div className={'col-auto ' + styles.heading}>
+              <h3>Client Feedback</h3>
+            </div>
+            <div className='col'></div>
+            <div className={'col-auto ' + styles.dots}>
+              <ul>
+                <li>
+                  <a className={styles.active}>page</a>
+                </li>
+                <li>
+                  <a>page</a>
+                </li>
+                <li>
+                  <a>page</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+        <div className='row justify-content-center mb-3'>
+          <img
```

Comment:
rozmawialiśmy żeby to zrobić jako font :P

Reply:
Poprawione.